src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -783,6 +788,16 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                                 emitInsHelper(targetReg, maskReg, embMaskOp2Reg);
                                 break;
 
+                            case NI_Sve2_ConvertToSingleOdd:
+                            case NI_Sve2_ConvertToSingleOddRoundToOdd:
+                                // TODO-SVE: Optimise away the explicit copying of `embMaskOp1Reg` to `targetReg`.
+                                // For these intrinsics we cannot use movprfx instruction to populate `targetReg` with
+                                // `embMaskOp1Reg`. Thus, we need to perform move before the operation.
+                                GetEmitter()->emitIns_R_R_R(INS_sve_orr, EA_SCALABLE, targetReg, embMaskOp1Reg,
```

Comment:
Switch to: `GetEmitter()->emitIns_Mov(INS_sve_mov, emitSize, targetReg, embMaskOp1Reg, /* canSkip */ true);`

That'll ensure it optimises away if `targetReg == embMaskOp1Reg`